mp/src/game/client/momentum/client_events.cpp

```diff
@@ -43,6 +43,21 @@ bool CanQuit(void *pUnused)
     return true;
 }
 
+inline void OverrideConVarDefault(const char *pName, const char *pValue)
+{
+    const auto pVar = g_pCVar->FindVar(pName);
+
+    if (pVar)
+    {
+        if (FStrEq(pVar->GetDefault(), pVar->GetString()))
```

Comment:
Should this be negated? It's otherwise setting the value if it is the default?

Reply:
That's the point, if the cvar was changed by the user then it should stay as is.